File: TRIANGLE PATTERN 1/code7.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter your no. of rows : ");
	scanf("%d",&row);

	for(int i = 0; i < row ; i++){

		for(int j = 0 ; j <= i ;j++){

			if((i)%4== 0){

				printf("%d\t",i+1);

			}else if(i%2==1){				
				printf("0\t");
			}else {

	
				printf("1\t");
			}

		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/7/code7.java
import java.util.Scanner;

class NestedForDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter no. of rows : ");
		int row = s.nextInt();

		for(int i = 1; i <= row ; i++){

			for(int j = row; j >= 1 ; j--){

				System.out.print(j+"	");

			}
			System.out.println("\n");

		}

	}


}
